source/OutfitInfoDisplay.cpp

```diff
@@ -352,7 +352,7 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 	if(outfit.Ammo())
 	{
 		attributeLabels.emplace_back("ammo:");
-		attributeValues.emplace_back(outfit.Ammo()->Name());
+		attributeValues.emplace_back(outfit.Ammo()->TrueName());
```

Comment:
This is the name of the ammo as shown in the outfit details, as such it should use Name() and not TrueName().

Reply:
You forgot to do this one.

Reply:
done